includes/gateways/class-wpsms-gateway-aspsms.php

```
@@ -8,6 +8,7 @@ class aspsms extends \WP_SMS\Gateway
 {
     private $wsdl_link = "https://json.aspsms.com/";
     public $tariff = "https://aspsms.com";
+    public $documentUrl = 'https://wsms.io/docs/aspsms-gateway-setup';
```

PHPStan (2.2.7)

[error] Property WP_SMS\Gateway\aspsms::$documentUrl (bool) does not accept default value of type string.


(property.defaultValue)